documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -140,7 +140,7 @@ When you navigate around your application, SvelteKit reuses existing layout and
 <div>{@html data.content}</div>
 ```
 
-...then navigating from `/blog/my-short-post` to `/blog/my-long-post` won't cause the component to be destroyed and recreated. The `data` prop (and by extension `data.title` and `data.content`) will change, but because the code isn't rerunning, `estimatedReadingTime` won't be recalculated.
+...then navigating from `/blog/my-short-post` to `/blog/my-long-post` won't cause the components (layout, page and any components on the page) to be destroyed and recreated. Instead the `data` prop (and by extension `data.title` and `data.content`) will reactively change (as it would with a regular Svelte component) and, because the code isn't rerunning, `estimatedReadingTime` won't be recalculated nor will lifecycle methods like `onMount` and `onDestroy` run.
```

Comment:
This isn't 100% correct, as we don't know how downstream components interact with each other - maybe an if block makes a different component appear? I'm not sure how to best phrase it to remove that ambiguity.

Reply:
```diff
- won't be recalculated nor will lifecycle methods like `onMount` and `onDestroy` run.
+ won't be recalculated and lifecycle methods like `onMount` and `onDestroy` won't rerun.

Reply:
I removed some of the text and elaborated more in the part where the example with a `#key` block is to emphasize that components that have already been rendered are not re-rendered.